django-ydb-backend/pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[project]
+name = "django-ydb-backend"
+version = "0.1.0"
+description = ""
+authors = [
+    {name = "Your Name",email = "you@example.com"}
+]
+readme = "README.md"
+requires-python = ">=3.13"
```

Comment:
let's start with some lower version (better check for other connectors or django's min python requirements

Reply:
also it is good to add <4